core/templates/services/literally-canvas-helper.service.ts

```diff
@@ -272,13 +256,13 @@ angular.module('oppia').factory('LiterallyCanvasHelperService', [
         var rect = doc.querySelector('svg > rect');
         snapshot.colors.primary = lc.colors.primary;
         snapshot.colors.secondary = lc.colors.secondary;
-        // @ts-ignore fill property doesnot exist error.
+        // @ts-ignore fill property does not exist error.
```

Comment:
Per @nishantwrp's [comment](https://github.com/oppia/oppia/pull/9445#discussion_r437008885), we shouldn't  be using ts-ignore here.

Reply:
Done.